### PR TITLE
Cleanup source tree test updates

### PR DESCRIPTION
**Pull request by darkwing**
Fixes: #6441

We need to get rid of the `source.get ? source.get('url') : source.url` stuff.

### CODE REVIEW

**Comment by darkwing** [comment]
@lukaszsobek I know you've worked with the immuatables and getting rid of `get`; is there something I'm missing?

File: src/utils/sources-tree/addToTree.js [hunk 1]
```diff
@@ -169,7 +169,13 @@ export function addToTree(
   debuggeeUrl: string,
   projectRoot: string
 ) {
-  const url = getURL(source.get ? source.get("url") : source.url, debuggeeUrl);
+
+  //console.log(source.url);
+  if (!source.url) {
+    console.log('NOOOOOOO',  source, source.url, source.get("url"));
+  }
+
```
Comment (jasonLaster) [inline src/utils/sources-tree/addToTree.js]: 😛

File: src/utils/sources-tree/utils.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ export function isNotJavaScript(source: Object): boolean {
 export function isInvalidUrl(url: Object, source: SourceRecord) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
-    !(source.get ? source.get("url") : source.url) ||
+    !source.get("url") ||
```
Comment (jasonLaster) [inline src/utils/sources-tree/utils.js]: we should be able to do `source.url` now

**Comment by lukaszsobek** [comment]
btw, according to the docs the dot notation only works for I.records, not for I.maps.

**Comment by darkwing** [comment]
w00t; changing `mergeIn` to `setIn` was the key!

**Pull request merged**